src/database/sql_database.cc

```diff
@@ -299,6 +299,21 @@ static const std::vector<std::pair<std::string, AutoscanColumn>> autoscanTagMap
     { "obj_location", AutoscanColumn::ObjLocation },
 };
 
+// Format string for a recursive query of a parent container
+static constexpr auto sql_search_container_query_raw = R"(
+-- Find all children of parent_id
+WITH {0} AS (SELECT * FROM {2} WHERE {4} = {{}}
+UNION
+SELECT {3}.* FROM {2} JOIN {0} AS {1} ON {3}.{4} = {1}.{5}
+),
+-- Find all physical items and de-reference any virtual item (i.e. follow ref-id)
+items AS (SELECT * from {0} AS {1} WHERE {6} IS NULL
+UNION
+SELECT {3}.* FROM {0} AS {1} JOIN {2} ON {1}.{6} = {3}.{5}
+)
+-- Select desired cols from items
```

Comment:
DId you check `--` comments in statements for mysql and sqlite APIs?

Reply:
Yes it looks like [MySQL](https://dev.mysql.com/doc/refman/5.7/en/comments.html) and [SQLite](https://www.sqlite.org/lang_comment.html) support `--` as comments